resources/web/wwi/video_manager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+function VideoManager(parentObject, mouseEvents, stream) {
```

Comment:
Same comment here, I would remove the `Manager` keyword and call this class simply `Video` and remove `video` from the method names.